Dockerfile

```diff
@@ -219,11 +219,18 @@ COPY . /work
 WORKDIR /work
 RUN ./pkg/rpm/build.sh
 
-FROM opensuse/leap:15.3 AS sles15-build
-
-RUN set -x; zypper -n install git systemd autoconf automake flex libtool libcurl-devel libopenssl-devel libyajl-devel gcc gcc-c++ zlib-devel rpm-build expect cmake systemd-devel systemd-rpm-macros java-11-openjdk-devel unzip zip && \
-    # Add home:d4vid:co22 repo to install >3.4 bison
-    zypper addrepo https://download.opensuse.org/repositories/home:/d4vid:/co22/15.3/home:d4vid:co22.repo && \
+FROM opensuse/leap:15.1 AS sles15-build
+SHELL ["/bin/bash", "-c"]
```

Comment:
Why do we need bash?

Reply:
docker by default why not specified, it uses `/bin/sh`. I got the solution from https://stackoverflow.com/a/33439625/8514230 which uses `/bin/bash`

Reply:
This should be doable on `/bin/sh`, e.g. https://stackoverflow.com/a/59245613/1188632